lib/utils/parse.go

```diff
@@ -419,3 +419,29 @@ func ToUnknownResource(resource teleservices.Resource) (*teleservices.UnknownRes
 	}
 	return &unknown, nil
 }
+
+// ParseProxyAddr parses proxy address in the format "host:webPort,sshPort"
+//
+// If web/SSH ports are missing the provided defaults are used.
+func ParseProxyAddr(proxyAddr, defaultWebPort, defaultSSHPort string) (host string, webPort string, sshPort string, err error) {
+	host, port, err := net.SplitHostPort(proxyAddr)
+	if err != nil {
+		return proxyAddr, defaultWebPort, defaultSSHPort, nil
```

Comment:
isn't this kind of dangerous - since the error is suppressed and not evaluated? Or is the condition flipped?